geoips/xarray_utils/xr_to_dtree.py

```diff
@@ -0,0 +1,27 @@
+# # # Distribution Statement A. Approved for public release. Distribution unlimited.
+# # #
+# # # Author:
+# # # Naval Research Laboratory, Marine Meteorology Division
+# # #
+# # # This program is free software: you can redistribute it and/or modify it under
+# # # the terms of the NRLMMD License included with this program. This program is
+# # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
+# # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
+# # # for more details. If you did not receive the license, for more information see:
+# # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+
+"""Takes in a dictionary of xarrays and converts to xarray datatree"""
+
+
+import logging
+from datatree import DataTree
+
+LOG = logging.getLogger(__name__)
+
+
+def xarray_to_datatree(xarray_dict):
+    """Convert a flat (non nested) dictionary of xarrays to DataTree format"""
+    xarray_datatree = DataTree.from_dict(xarray_dict)

```

Comment:
wow, this is pretty straightforward, huh...

Reply:
Yes, I could add a check for a nested dictionary of xarrays if needed...?

Reply:
@mindyls Do we ever have tested dictionaries of xarrays? I hope not!

Reply:
I don't think we do.  @AJThorpe I think we might need to do something to transfer the metadata that is currently stored in the "METADATA" dictionary key, because I expect that could be stored at the top level of the datatree directly, rather than in a  full separate xarray Dataset?

Do you want to see what that might look like?

Reply:
Yes, understood, I'll add that as the root group in the code

Reply:
@AJThorpe Did you make this update?